frontend/amundsen_application/static/css/_layouts.scss

```diff
@@ -26,6 +26,8 @@ $icon-header-size: 32px;
     }
 
     .header-section {
+      flex-shrink: 0;
```

Comment:
context for the change of this line: https://github.com/amundsen-io/amundsen/pull/1855

it has been merged into upstream.